builder/builder_test.go

```diff
@@ -208,7 +211,7 @@ func TestRollback(t *testing.T) {
 		require.NoError(t, eventBus.Stop())
 	})
 
-	require.NoError(t, g.Commit(context.Background(), app, blockStore))
+	require.NoError(t, g.Commit(context.Background(), app, blockStore, ethstatedb))
```

Comment:
Let's add todos to add more tests to ensure we're actually updating the withdrawals trie.